File: WaterManagerUI/Model/Security/LoginResponseDTO.cs
using Newtonsoft.Json;
using UserInterfaceWaterManager.Model.User;

namespace WaterManagerUI.Model.Security;

public class LoginResponseDTO
{
    public UserProfile user { get; set; }
    public String jwt { get; set; }

    [JsonConstructor]
    public LoginResponseDTO(UserProfile user, string jwt)
    {
        this.user = user;
        this.jwt = jwt;
    }

    public LoginResponseDTO()
    {
    }
}

File: WaterManager/src/main/java/pissir/watermanager/security/model/LoginResponseDTO.java
package pissir.watermanager.security.model;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import pissir.watermanager.model.user.UserProfile;

/**
 * @author Almasio Luca
 * @author Borova Dritan
 * @author Gattico Alessandro
 */

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class LoginResponseDTO {
	private UserProfile user;
	private String jwt;
}
